frodo.js

```diff
@@ -5,47 +5,51 @@
  * for the extras/ plugins if they are loaded into a presentation.
```

Comment:
SCRIPT-Consider using more descriptive naming for this boolean variable to clarify its purpose.